---
result: FAILURE
url: http://wilson.ci.chef.co/job/chef-server-12-trigger-ad_hoc/114/
failure_category: code
failure_cause: 'chef-client: all: chef-server-12-test'
timestamp: 2016-03-21 18:05:06 UTC
duration: 1h32m43s
triggered_by: jmink
active_duration: 1h32m28s
parameters:
  GIT_REF: SPOOL-92/update_inet_interface
  EXPIRE_CACHE: false
change:
  git_remote: git@github.com:chef/chef-server.git
  git_commit: f965535e77e34a1903b8befc2749fc9cf40abeee
  project: chef-server
  version: 12.4.2+20160321180522
stages:
  chef-server-12-test:
    result: FAILURE
    failure_category: code
    failure_cause: 'chef-client: all'
    url: http://wilson.ci.chef.co/job/chef-server-12-test/156/
    duration: 57m52s
    runs:
      el-5:
        result: FAILURE
        failure_category: code
        failure_cause: chef-client
        failed_in:
          step: chef-client
        url: http://wilson.ci.chef.co/job/chef-server-12-test/architecture=x86_64,platform=el-5,project=chef-server,role=tester/156/
        duration: 10m53s
        logs:
          consoleText:
            1558: |
              \33[0mCookbook Trace:\33[0m
              ---------------\33[0m
              /opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/libraries/chef_server_data_bootstrap.rb:187:in `bifrost_request'
              \33[0m/opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/libraries/chef_server_data_bootstrap.rb:154:in `create_object_in_authz'
              \33[0m/opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/libraries/chef_server_data_bootstrap.rb:146:in `create_actor_in_authz'
              \33[0m/opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/libraries/chef_server_data_bootstrap.rb:36:in `bootstrap'
              \33[0m/opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/recipes/bootstrap.rb:46:in `block (2 levels) in from_file'\33[0m

              \33[0mResource Declaration:\33[0m
            1661: |
              \33[0mCookbook Trace:\33[0m
              ---------------\33[0m
              /opt/opscode/embedded/cookbooks/cache/cookbooks/runit/libraries/provider_runit_service.rb:184:in `restart_service'\33[0m

              \33[0mResource Declaration:\33[0m
            1757: |
              [2016-03-21T15:35:39-04:00] INFO: Processing execute[restart_opscode-erchef_log_service] action run (private-chef::opscode-erchef line 19)
              [2016-03-21T15:35:39-04:00] INFO: execute[restart_opscode-erchef_log_service] ran successfully
              [2016-03-21T15:35:39-04:00] ERROR: Running exception handlers
              [2016-03-21T15:35:39-04:00] ERROR: Exception handlers complete
              [2016-03-21T15:35:39-04:00] FATAL: Stacktrace dumped to /opt/opscode/embedded/cookbooks/cache/chef-stacktrace.out
              [2016-03-21T15:35:39-04:00] FATAL: Please provide the contents of the stacktrace.out file if you file a bug report
              [2016-03-21T15:35:40-04:00] FATAL: Chef::Exceptions::MultipleFailures: Multiple failures occurred:
              * Errno::ECONNREFUSED occurred in chef run: ruby_block[bootstrap-chef-server-data] (private-chef::bootstrap line 44) had an error: Errno::ECONNREFUSED: Connection refused - connect(2) for "127.0.0.1" port 9463
              * Mixlib::ShellOut::ShellCommandFailed occurred in delayed notification: runit_service[nginx] (private-chef::nginx line 37) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '1'
            1782: |
              No entry for terminal type "unknown";
              using dumb terminal settings.
              /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant/config.rb:34:in `from_argv': Configuration file '/var/opt/opscode/oc-chef-pedant/etc/pedant_config.rb' not found! (RuntimeError)
                from /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant.rb:54:in `setup'
                from ./bin/oc-chef-pedant:10:in `<main>'
              Build step 'Invoke XShell command' marked build as failure
              Performing Post build task...
              Match found for : : True
            1806: |
              Recording fingerprints
              Recording test results
              ERROR: Publisher hudson.tasks.junit.JUnitResultArchiver aborted due to exception
              hudson.AbortException: No test report files were found. Configuration error?
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:116)
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:92)
                at hudson.FilePath$FileCallableWrapper.call(FilePath.java:2688)
                at hudson.remoting.UserRequest.perform(UserRequest.java:121)
                at hudson.remoting.UserRequest.perform(UserRequest.java:49)
                at hudson.remoting.Request$2.run(Request.java:325)
                at hudson.remoting.InterceptingExecutorService$1.call(InterceptingExecutorService.java:68)
                at java.util.concurrent.FutureTask.run(FutureTask.java:262)
                at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
                at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
                at java.lang.Thread.run(Thread.java:745)
                at ......remote call to chef-server-rhel-5-tester-4e6602(Native Method)
                at hudson.remoting.Channel.attachCallSiteStackTrace(Channel.java:1360)
                at hudson.remoting.UserResponse.retrieve(UserRequest.java:221)
                at hudson.remoting.Channel.call(Channel.java:753)
                at hudson.FilePath.act(FilePath.java:978)
                at hudson.FilePath.act(FilePath.java:967)
                at hudson.tasks.junit.JUnitParser.parseResult(JUnitParser.java:89)
                at hudson.tasks.junit.JUnitResultArchiver.parse(JUnitResultArchiver.java:121)
                at hudson.tasks.junit.JUnitResultArchiver.perform(JUnitResultArchiver.java:138)
                at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:74)
                at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
                at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:761)
                at hudson.model.AbstractBuild$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:721)
                at hudson.model.Build$BuildExecution.post2(Build.java:183)
                at hudson.model.AbstractBuild$AbstractBuildExecution.post(AbstractBuild.java:670)
                at hudson.model.Run.execute(Run.java:1766)
                at hudson.matrix.MatrixRun.run(MatrixRun.java:146)
                at hudson.model.ResourceController.execute(ResourceController.java:98)
                at hudson.model.Executor.run(Executor.java:374)
              Regular expression run condition: Expression=[^(?!true)$], Label=[true]
              Run condition [Regular expression match] preventing perform for step [[Editable Email Notification]]
        steps:
          total: 10m53s
          before (setup time): 51m25s
          after (cleanup time): "-40m34s"
      el-6:
        result: FAILURE
        failure_category: code
        failure_cause: chef-client
        failed_in:
          step: chef-client
        url: http://wilson.ci.chef.co/job/chef-server-12-test/architecture=x86_64,platform=el-6,project=chef-server,role=tester/156/
        duration: 8m50s
        logs:
          consoleText:
            1544: |
              \33[0mCookbook Trace:\33[0m
              ---------------\33[0m
              /opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/libraries/chef_server_data_bootstrap.rb:187:in `bifrost_request'
              \33[0m/opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/libraries/chef_server_data_bootstrap.rb:154:in `create_object_in_authz'
              \33[0m/opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/libraries/chef_server_data_bootstrap.rb:146:in `create_actor_in_authz'
              \33[0m/opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/libraries/chef_server_data_bootstrap.rb:36:in `bootstrap'
              \33[0m/opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/recipes/bootstrap.rb:46:in `block (2 levels) in from_file'\33[0m

              \33[0mResource Declaration:\33[0m
            1647: |
              \33[0mCookbook Trace:\33[0m
              ---------------\33[0m
              /opt/opscode/embedded/cookbooks/cache/cookbooks/runit/libraries/provider_runit_service.rb:184:in `restart_service'\33[0m

              \33[0mResource Declaration:\33[0m
            1743: |
              [2016-03-21T15:06:19-04:00] INFO: Processing execute[restart_opscode-erchef_log_service] action run (private-chef::opscode-erchef line 19)
              [2016-03-21T15:06:19-04:00] INFO: execute[restart_opscode-erchef_log_service] ran successfully
              [2016-03-21T15:06:19-04:00] ERROR: Running exception handlers
              [2016-03-21T15:06:19-04:00] ERROR: Exception handlers complete
              [2016-03-21T15:06:19-04:00] FATAL: Stacktrace dumped to /opt/opscode/embedded/cookbooks/cache/chef-stacktrace.out
              [2016-03-21T15:06:19-04:00] FATAL: Please provide the contents of the stacktrace.out file if you file a bug report
              [2016-03-21T15:06:19-04:00] FATAL: Chef::Exceptions::MultipleFailures: Multiple failures occurred:
              * Errno::ECONNREFUSED occurred in chef run: ruby_block[bootstrap-chef-server-data] (private-chef::bootstrap line 44) had an error: Errno::ECONNREFUSED: Connection refused - connect(2) for "127.0.0.1" port 9463
              * Mixlib::ShellOut::ShellCommandFailed occurred in delayed notification: runit_service[nginx] (private-chef::nginx line 37) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '1'
            1768: |
              No entry for terminal type "unknown";
              using dumb terminal settings.
              /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant/config.rb:34:in `from_argv': Configuration file '/var/opt/opscode/oc-chef-pedant/etc/pedant_config.rb' not found! (RuntimeError)
                from /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant.rb:54:in `setup'
                from ./bin/oc-chef-pedant:10:in `<main>'
              Build step 'Invoke XShell command' marked build as failure
              Performing Post build task...
              Match found for : : True
            1792: |
              Recording fingerprints
              Recording test results
              ERROR: Publisher hudson.tasks.junit.JUnitResultArchiver aborted due to exception
              hudson.AbortException: No test report files were found. Configuration error?
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:116)
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:92)
                at hudson.FilePath$FileCallableWrapper.call(FilePath.java:2688)
                at hudson.remoting.UserRequest.perform(UserRequest.java:121)
                at hudson.remoting.UserRequest.perform(UserRequest.java:49)
                at hudson.remoting.Request$2.run(Request.java:325)
                at hudson.remoting.InterceptingExecutorService$1.call(InterceptingExecutorService.java:68)
                at java.util.concurrent.FutureTask.run(FutureTask.java:262)
                at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
                at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
                at java.lang.Thread.run(Thread.java:745)
                at ......remote call to chef-server-rhel-6-tester-8151c8(Native Method)
                at hudson.remoting.Channel.attachCallSiteStackTrace(Channel.java:1360)
                at hudson.remoting.UserResponse.retrieve(UserRequest.java:221)
                at hudson.remoting.Channel.call(Channel.java:753)
                at hudson.FilePath.act(FilePath.java:978)
                at hudson.FilePath.act(FilePath.java:967)
                at hudson.tasks.junit.JUnitParser.parseResult(JUnitParser.java:89)
                at hudson.tasks.junit.JUnitResultArchiver.parse(JUnitResultArchiver.java:121)
                at hudson.tasks.junit.JUnitResultArchiver.perform(JUnitResultArchiver.java:138)
                at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:74)
                at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
                at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:761)
                at hudson.model.AbstractBuild$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:721)
                at hudson.model.Build$BuildExecution.post2(Build.java:183)
                at hudson.model.AbstractBuild$AbstractBuildExecution.post(AbstractBuild.java:670)
                at hudson.model.Run.execute(Run.java:1766)
                at hudson.matrix.MatrixRun.run(MatrixRun.java:146)
                at hudson.model.ResourceController.execute(ResourceController.java:98)
                at hudson.model.Executor.run(Executor.java:374)
              Regular expression run condition: Expression=[^(?!true)$], Label=[true]
              Run condition [Regular expression match] preventing perform for step [[Editable Email Notification]]
        steps:
          total: 8m50s
          before (setup time): 22m18s
          after (cleanup time): "-13m31s"
      el-7:
        result: FAILURE
        failure_category: code
        failure_cause: chef-client
        failed_in:
          step: chef-client
        url: http://wilson.ci.chef.co/job/chef-server-12-test/architecture=x86_64,platform=el-7,project=chef-server,role=tester/156/
        duration: 7m48s
        logs:
          consoleText:
            51221: |
              warning: file /opt/opscode/embedded/lib/ruby/gems/2.1.0/doc/nokogiri-1.6.7.2/ri/Nokogiri/XML/SyntaxError/int1-i.ri: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/ruby/gems/2.1.0/doc/nokogiri-1.6.7.2/ri/Nokogiri/XML/SyntaxError/file-i.ri: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/ruby/gems/2.1.0/doc/nokogiri-1.6.7.2/ri/Nokogiri/XML/SyntaxError/fatal%3f-i.ri: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/ruby/gems/2.1.0/doc/nokogiri-1.6.7.2/ri/Nokogiri/XML/SyntaxError/error%3f-i.ri: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/ruby/gems/2.1.0/doc/nokogiri-1.6.7.2/ri/Nokogiri/XML/SyntaxError/domain-i.ri: remove failed: No such file or directory
            57811: |
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/auto/Test/File/.packlist: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/auto/Test/File: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/auto/Test/Fatal/.packlist: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/auto/Test/Fatal: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/auto/Test/Exception/.packlist: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/auto/Test/Exception: remove failed: No such file or directory
            58757: |
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-File-1.44/MYMETA.json: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-File-1.44: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-Fatal-0.014/install.json: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-Fatal-0.014/MYMETA.json: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-Fatal-0.014: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-Exception-0.43/install.json: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-Exception-0.43/MYMETA.json: remove failed: No such file or directory
            59139: |
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/Test/File.pm: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/Test/File: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/Test/Fatal.pm: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/Test/Exception.pm: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/Test/Dir/Base.pm: remove failed: No such file or directory
            62958: |
              warning: file /opt/opscode/embedded/lib/perl5/5.18.1/File/Basename.pm: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/5.18.1/File: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/5.18.1/Fatal.pm: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/5.18.1/ExtUtils/xsubpp: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/5.18.1/ExtUtils/typemap: remove failed: No such file or directory
            73542: |
              \33[0mCookbook Trace:\33[0m
              ---------------\33[0m
              /opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/libraries/chef_server_data_bootstrap.rb:187:in `bifrost_request'
              \33[0m/opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/libraries/chef_server_data_bootstrap.rb:154:in `create_object_in_authz'
              \33[0m/opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/libraries/chef_server_data_bootstrap.rb:146:in `create_actor_in_authz'
              \33[0m/opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/libraries/chef_server_data_bootstrap.rb:36:in `bootstrap'
              \33[0m/opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/recipes/bootstrap.rb:46:in `block (2 levels) in from_file'\33[0m

              \33[0mResource Declaration:\33[0m
            73645: |
              \33[0mCookbook Trace:\33[0m
              ---------------\33[0m
              /opt/opscode/embedded/cookbooks/cache/cookbooks/runit/libraries/provider_runit_service.rb:184:in `restart_service'\33[0m

              \33[0mResource Declaration:\33[0m
            73741: |
              [2016-03-21T14:48:54-04:00] INFO: Processing execute[restart_opscode-erchef_log_service] action run (private-chef::opscode-erchef line 19)
              [2016-03-21T14:48:55-04:00] INFO: execute[restart_opscode-erchef_log_service] ran successfully
              [2016-03-21T14:48:55-04:00] ERROR: Running exception handlers
              [2016-03-21T14:48:55-04:00] ERROR: Exception handlers complete
              [2016-03-21T14:48:55-04:00] FATAL: Stacktrace dumped to /opt/opscode/embedded/cookbooks/cache/chef-stacktrace.out
              [2016-03-21T14:48:55-04:00] FATAL: Please provide the contents of the stacktrace.out file if you file a bug report
              [2016-03-21T14:48:55-04:00] FATAL: Chef::Exceptions::MultipleFailures: Multiple failures occurred:
              * Errno::ECONNREFUSED occurred in chef run: ruby_block[bootstrap-chef-server-data] (private-chef::bootstrap line 44) had an error: Errno::ECONNREFUSED: Connection refused - connect(2) for "127.0.0.1" port 9463
              * Mixlib::ShellOut::ShellCommandFailed occurred in delayed notification: runit_service[nginx] (private-chef::nginx line 37) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '1'
            73766: |
              No entry for terminal type "unknown";
              using dumb terminal settings.
              /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant/config.rb:34:in `from_argv': Configuration file '/var/opt/opscode/oc-chef-pedant/etc/pedant_config.rb' not found! (RuntimeError)
                from /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant.rb:54:in `setup'
                from ./bin/oc-chef-pedant:10:in `<main>'
              Build step 'Invoke XShell command' marked build as failure
              Performing Post build task...
              Match found for : : True
            73790: |
              Recording fingerprints
              Recording test results
              ERROR: Publisher hudson.tasks.junit.JUnitResultArchiver aborted due to exception
              hudson.AbortException: No test report files were found. Configuration error?
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:116)
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:92)
                at hudson.FilePath$FileCallableWrapper.call(FilePath.java:2688)
                at hudson.remoting.UserRequest.perform(UserRequest.java:121)
                at hudson.remoting.UserRequest.perform(UserRequest.java:49)
                at hudson.remoting.Request$2.run(Request.java:325)
                at hudson.remoting.InterceptingExecutorService$1.call(InterceptingExecutorService.java:68)
                at java.util.concurrent.FutureTask.run(FutureTask.java:262)
                at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
                at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
                at java.lang.Thread.run(Thread.java:745)
                at ......remote call to chef-server-rhel-7-tester-d586ce(Native Method)
                at hudson.remoting.Channel.attachCallSiteStackTrace(Channel.java:1360)
                at hudson.remoting.UserResponse.retrieve(UserRequest.java:221)
                at hudson.remoting.Channel.call(Channel.java:753)
                at hudson.FilePath.act(FilePath.java:978)
                at hudson.FilePath.act(FilePath.java:967)
                at hudson.tasks.junit.JUnitParser.parseResult(JUnitParser.java:89)
                at hudson.tasks.junit.JUnitResultArchiver.parse(JUnitResultArchiver.java:121)
                at hudson.tasks.junit.JUnitResultArchiver.perform(JUnitResultArchiver.java:138)
                at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:74)
                at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
                at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:761)
                at hudson.model.AbstractBuild$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:721)
                at hudson.model.Build$BuildExecution.post2(Build.java:183)
                at hudson.model.AbstractBuild$AbstractBuildExecution.post(AbstractBuild.java:670)
                at hudson.model.Run.execute(Run.java:1766)
                at hudson.matrix.MatrixRun.run(MatrixRun.java:146)
                at hudson.model.ResourceController.execute(ResourceController.java:98)
                at hudson.model.Executor.run(Executor.java:374)
              Regular expression run condition: Expression=[^(?!true)$], Label=[true]
              Run condition [Regular expression match] preventing perform for step [[Editable Email Notification]]
        steps:
          total: 7m48s
          before (setup time): 6m27s
          after (cleanup time): 1m17s
      ubuntu-10.04:
        result: FAILURE
        failure_category: code
        failure_cause: chef-client
        failed_in:
          step: chef-client
        url: http://wilson.ci.chef.co/job/chef-server-12-test/architecture=x86_64,platform=ubuntu-10.04,project=chef-server,role=tester/156/
        duration: 7m32s
        logs:
          consoleText:
            1571: |
              \33[0mCookbook Trace:\33[0m
              ---------------\33[0m
              /opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/libraries/chef_server_data_bootstrap.rb:187:in `bifrost_request'
              \33[0m/opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/libraries/chef_server_data_bootstrap.rb:154:in `create_object_in_authz'
              \33[0m/opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/libraries/chef_server_data_bootstrap.rb:146:in `create_actor_in_authz'
              \33[0m/opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/libraries/chef_server_data_bootstrap.rb:36:in `bootstrap'
              \33[0m/opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/recipes/bootstrap.rb:46:in `block (2 levels) in from_file'\33[0m

              \33[0mResource Declaration:\33[0m
            1674: |
              \33[0mCookbook Trace:\33[0m
              ---------------\33[0m
              /opt/opscode/embedded/cookbooks/cache/cookbooks/runit/libraries/provider_runit_service.rb:184:in `restart_service'\33[0m

              \33[0mResource Declaration:\33[0m
            1770: |
              [2016-03-21T19:02:07+00:00] INFO: Processing execute[restart_opscode-erchef_log_service] action run (private-chef::opscode-erchef line 19)
              [2016-03-21T19:02:07+00:00] INFO: execute[restart_opscode-erchef_log_service] ran successfully
              [2016-03-21T19:02:07+00:00] ERROR: Running exception handlers
              [2016-03-21T19:02:07+00:00] ERROR: Exception handlers complete
              [2016-03-21T19:02:07+00:00] FATAL: Stacktrace dumped to /opt/opscode/embedded/cookbooks/cache/chef-stacktrace.out
              [2016-03-21T19:02:07+00:00] FATAL: Please provide the contents of the stacktrace.out file if you file a bug report
              [2016-03-21T19:02:07+00:00] FATAL: Chef::Exceptions::MultipleFailures: Multiple failures occurred:
              * Errno::ECONNREFUSED occurred in chef run: ruby_block[bootstrap-chef-server-data] (private-chef::bootstrap line 44) had an error: Errno::ECONNREFUSED: Connection refused - connect(2) for "127.0.0.1" port 9463
              * Mixlib::ShellOut::ShellCommandFailed occurred in delayed notification: runit_service[nginx] (private-chef::nginx line 37) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '1'
            1795: |
              No entry for terminal type "unknown";
              using dumb terminal settings.
              /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant/config.rb:34:in `from_argv': Configuration file '/var/opt/opscode/oc-chef-pedant/etc/pedant_config.rb' not found! (RuntimeError)
                from /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant.rb:54:in `setup'
                from ./bin/oc-chef-pedant:10:in `<main>'
              Build step 'Invoke XShell command' marked build as failure
              Performing Post build task...
              Match found for : : True
            1819: |
              Recording fingerprints
              Recording test results
              ERROR: Publisher hudson.tasks.junit.JUnitResultArchiver aborted due to exception
              hudson.AbortException: No test report files were found. Configuration error?
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:116)
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:92)
                at hudson.FilePath$FileCallableWrapper.call(FilePath.java:2688)
                at hudson.remoting.UserRequest.perform(UserRequest.java:121)
                at hudson.remoting.UserRequest.perform(UserRequest.java:49)
                at hudson.remoting.Request$2.run(Request.java:325)
                at hudson.remoting.InterceptingExecutorService$1.call(InterceptingExecutorService.java:68)
                at java.util.concurrent.FutureTask.run(FutureTask.java:262)
                at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
                at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
                at java.lang.Thread.run(Thread.java:745)
                at ......remote call to chef-server-ubuntu-10.04-tester-f80d1f(Native Method)
                at hudson.remoting.Channel.attachCallSiteStackTrace(Channel.java:1360)
                at hudson.remoting.UserResponse.retrieve(UserRequest.java:221)
                at hudson.remoting.Channel.call(Channel.java:753)
                at hudson.FilePath.act(FilePath.java:978)
                at hudson.FilePath.act(FilePath.java:967)
                at hudson.tasks.junit.JUnitParser.parseResult(JUnitParser.java:89)
                at hudson.tasks.junit.JUnitResultArchiver.parse(JUnitResultArchiver.java:121)
                at hudson.tasks.junit.JUnitResultArchiver.perform(JUnitResultArchiver.java:138)
                at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:74)
                at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
                at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:761)
                at hudson.model.AbstractBuild$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:721)
                at hudson.model.Build$BuildExecution.post2(Build.java:183)
                at hudson.model.AbstractBuild$AbstractBuildExecution.post(AbstractBuild.java:670)
                at hudson.model.Run.execute(Run.java:1766)
                at hudson.matrix.MatrixRun.run(MatrixRun.java:146)
                at hudson.model.ResourceController.execute(ResourceController.java:98)
                at hudson.model.Executor.run(Executor.java:374)
              Regular expression run condition: Expression=[^(?!true)$], Label=[true]
              Run condition [Regular expression match] preventing perform for step [[Editable Email Notification]]
        steps:
          total: 7m32s
          before (setup time): 19m7s
          after (cleanup time): "-11m38s"
      ubuntu-12.04:
        result: FAILURE
        failure_category: code
        failure_cause: chef-client
        failed_in:
          step: chef-client
        url: http://wilson.ci.chef.co/job/chef-server-12-test/architecture=x86_64,platform=ubuntu-12.04,project=chef-server,role=tester/156/
        duration: 7m36s
        logs:
          consoleText:
            1576: |
              \33[0mCookbook Trace:\33[0m
              ---------------\33[0m
              /opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/libraries/chef_server_data_bootstrap.rb:187:in `bifrost_request'
              \33[0m/opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/libraries/chef_server_data_bootstrap.rb:154:in `create_object_in_authz'
              \33[0m/opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/libraries/chef_server_data_bootstrap.rb:146:in `create_actor_in_authz'
              \33[0m/opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/libraries/chef_server_data_bootstrap.rb:36:in `bootstrap'
              \33[0m/opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/recipes/bootstrap.rb:46:in `block (2 levels) in from_file'\33[0m

              \33[0mResource Declaration:\33[0m
            1679: |
              \33[0mCookbook Trace:\33[0m
              ---------------\33[0m
              /opt/opscode/embedded/cookbooks/cache/cookbooks/runit/libraries/provider_runit_service.rb:184:in `restart_service'\33[0m

              \33[0mResource Declaration:\33[0m
            1775: |
              [2016-03-21T18:56:15+00:00] INFO: Processing execute[restart_opscode-erchef_log_service] action run (private-chef::opscode-erchef line 19)
              [2016-03-21T18:56:15+00:00] INFO: execute[restart_opscode-erchef_log_service] ran successfully
              [2016-03-21T18:56:15+00:00] ERROR: Running exception handlers
              [2016-03-21T18:56:15+00:00] ERROR: Exception handlers complete
              [2016-03-21T18:56:15+00:00] FATAL: Stacktrace dumped to /opt/opscode/embedded/cookbooks/cache/chef-stacktrace.out
              [2016-03-21T18:56:15+00:00] FATAL: Please provide the contents of the stacktrace.out file if you file a bug report
              [2016-03-21T18:56:15+00:00] FATAL: Chef::Exceptions::MultipleFailures: Multiple failures occurred:
              * Errno::ECONNREFUSED occurred in chef run: ruby_block[bootstrap-chef-server-data] (private-chef::bootstrap line 44) had an error: Errno::ECONNREFUSED: Connection refused - connect(2) for "127.0.0.1" port 9463
              * Mixlib::ShellOut::ShellCommandFailed occurred in delayed notification: runit_service[nginx] (private-chef::nginx line 37) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '1'
            1800: |
              No entry for terminal type "unknown";
              using dumb terminal settings.
              /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant/config.rb:34:in `from_argv': Configuration file '/var/opt/opscode/oc-chef-pedant/etc/pedant_config.rb' not found! (RuntimeError)
                from /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant.rb:54:in `setup'
                from ./bin/oc-chef-pedant:10:in `<main>'
              Build step 'Invoke XShell command' marked build as failure
              Performing Post build task...
              Match found for : : True
            1824: |
              Recording fingerprints
              Recording test results
              ERROR: Publisher hudson.tasks.junit.JUnitResultArchiver aborted due to exception
              hudson.AbortException: No test report files were found. Configuration error?
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:116)
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:92)
                at hudson.FilePath$FileCallableWrapper.call(FilePath.java:2688)
                at hudson.remoting.UserRequest.perform(UserRequest.java:121)
                at hudson.remoting.UserRequest.perform(UserRequest.java:49)
                at hudson.remoting.Request$2.run(Request.java:325)
                at hudson.remoting.InterceptingExecutorService$1.call(InterceptingExecutorService.java:68)
                at java.util.concurrent.FutureTask.run(FutureTask.java:262)
                at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
                at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
                at java.lang.Thread.run(Thread.java:745)
                at ......remote call to chef-server-ubuntu-12.04-tester-28c735(Native Method)
                at hudson.remoting.Channel.attachCallSiteStackTrace(Channel.java:1360)
                at hudson.remoting.UserResponse.retrieve(UserRequest.java:221)
                at hudson.remoting.Channel.call(Channel.java:753)
                at hudson.FilePath.act(FilePath.java:978)
                at hudson.FilePath.act(FilePath.java:967)
                at hudson.tasks.junit.JUnitParser.parseResult(JUnitParser.java:89)
                at hudson.tasks.junit.JUnitResultArchiver.parse(JUnitResultArchiver.java:121)
                at hudson.tasks.junit.JUnitResultArchiver.perform(JUnitResultArchiver.java:138)
                at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:74)
                at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
                at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:761)
                at hudson.model.AbstractBuild$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:721)
                at hudson.model.Build$BuildExecution.post2(Build.java:183)
                at hudson.model.AbstractBuild$AbstractBuildExecution.post(AbstractBuild.java:670)
                at hudson.model.Run.execute(Run.java:1766)
                at hudson.matrix.MatrixRun.run(MatrixRun.java:146)
                at hudson.model.ResourceController.execute(ResourceController.java:98)
                at hudson.model.Executor.run(Executor.java:374)
              Regular expression run condition: Expression=[^(?!true)$], Label=[true]
              Run condition [Regular expression match] preventing perform for step [[Editable Email Notification]]
        steps:
          total: 7m36s
          before (setup time): 13m45s
          after (cleanup time): "-6m10s"
      ubuntu-14.04:
        result: FAILURE
        failure_category: code
        failure_cause: chef-client
        failed_in:
          step: chef-client
        url: http://wilson.ci.chef.co/job/chef-server-12-test/architecture=x86_64,platform=ubuntu-14.04,project=chef-server,role=tester/156/
        duration: 7m17s
        logs:
          consoleText:
            1599: |
              \33[0mCookbook Trace:\33[0m
              ---------------\33[0m
              /opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/libraries/chef_server_data_bootstrap.rb:187:in `bifrost_request'
              \33[0m/opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/libraries/chef_server_data_bootstrap.rb:154:in `create_object_in_authz'
              \33[0m/opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/libraries/chef_server_data_bootstrap.rb:146:in `create_actor_in_authz'
              \33[0m/opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/libraries/chef_server_data_bootstrap.rb:36:in `bootstrap'
              \33[0m/opt/opscode/embedded/cookbooks/cache/cookbooks/private-chef/recipes/bootstrap.rb:46:in `block (2 levels) in from_file'\33[0m

              \33[0mResource Declaration:\33[0m
            1702: |
              \33[0mCookbook Trace:\33[0m
              ---------------\33[0m
              /opt/opscode/embedded/cookbooks/cache/cookbooks/runit/libraries/provider_runit_service.rb:184:in `restart_service'\33[0m

              \33[0mResource Declaration:\33[0m
            1798: |
              [2016-03-21T19:00:14+00:00] INFO: Processing execute[restart_opscode-erchef_log_service] action run (private-chef::opscode-erchef line 19)
              [2016-03-21T19:00:14+00:00] INFO: execute[restart_opscode-erchef_log_service] ran successfully
              [2016-03-21T19:00:14+00:00] ERROR: Running exception handlers
              [2016-03-21T19:00:14+00:00] ERROR: Exception handlers complete
              [2016-03-21T19:00:14+00:00] FATAL: Stacktrace dumped to /opt/opscode/embedded/cookbooks/cache/chef-stacktrace.out
              [2016-03-21T19:00:14+00:00] FATAL: Please provide the contents of the stacktrace.out file if you file a bug report
              [2016-03-21T19:00:14+00:00] FATAL: Chef::Exceptions::MultipleFailures: Multiple failures occurred:
              * Errno::ECONNREFUSED occurred in chef run: ruby_block[bootstrap-chef-server-data] (private-chef::bootstrap line 44) had an error: Errno::ECONNREFUSED: Connection refused - connect(2) for "127.0.0.1" port 9463
              * Mixlib::ShellOut::ShellCommandFailed occurred in delayed notification: runit_service[nginx] (private-chef::nginx line 37) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '1'
            1823: |
              No entry for terminal type "unknown";
              using dumb terminal settings.
              /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant/config.rb:34:in `from_argv': Configuration file '/var/opt/opscode/oc-chef-pedant/etc/pedant_config.rb' not found! (RuntimeError)
                from /opt/opscode/embedded/service/oc-chef-pedant/lib/pedant.rb:54:in `setup'
                from ./bin/oc-chef-pedant:10:in `<main>'
              Build step 'Invoke XShell command' marked build as failure
              Performing Post build task...
              Match found for : : True
            1847: |
              Recording fingerprints
              Recording test results
              ERROR: Publisher hudson.tasks.junit.JUnitResultArchiver aborted due to exception
              hudson.AbortException: No test report files were found. Configuration error?
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:116)
                at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:92)
                at hudson.FilePath$FileCallableWrapper.call(FilePath.java:2688)
                at hudson.remoting.UserRequest.perform(UserRequest.java:121)
                at hudson.remoting.UserRequest.perform(UserRequest.java:49)
                at hudson.remoting.Request$2.run(Request.java:325)
                at hudson.remoting.InterceptingExecutorService$1.call(InterceptingExecutorService.java:68)
                at java.util.concurrent.FutureTask.run(FutureTask.java:262)
                at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
                at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
                at java.lang.Thread.run(Thread.java:745)
                at ......remote call to chef-server-ubuntu-14.04-tester-69fc04(Native Method)
                at hudson.remoting.Channel.attachCallSiteStackTrace(Channel.java:1360)
                at hudson.remoting.UserResponse.retrieve(UserRequest.java:221)
                at hudson.remoting.Channel.call(Channel.java:753)
                at hudson.FilePath.act(FilePath.java:978)
                at hudson.FilePath.act(FilePath.java:967)
                at hudson.tasks.junit.JUnitParser.parseResult(JUnitParser.java:89)
                at hudson.tasks.junit.JUnitResultArchiver.parse(JUnitResultArchiver.java:121)
                at hudson.tasks.junit.JUnitResultArchiver.perform(JUnitResultArchiver.java:138)
                at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:74)
                at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
                at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:761)
                at hudson.model.AbstractBuild$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:721)
                at hudson.model.Build$BuildExecution.post2(Build.java:183)
                at hudson.model.AbstractBuild$AbstractBuildExecution.post(AbstractBuild.java:670)
                at hudson.model.Run.execute(Run.java:1766)
                at hudson.matrix.MatrixRun.run(MatrixRun.java:146)
                at hudson.model.ResourceController.execute(ResourceController.java:98)
                at hudson.model.Executor.run(Executor.java:374)
              Regular expression run condition: Expression=[^(?!true)$], Label=[true]
              Run condition [Regular expression match] preventing perform for step [[Editable Email Notification]]
        steps:
          total: 7m17s
          before (setup time): 17m26s
          after (cleanup time): "-10m10s"
  chef-server-12-build:
    result: SUCCESS
    url: http://wilson.ci.chef.co/job/chef-server-12-build/191/
    duration: 34m26s
    runs:
      el-5:
        result: SUCCESS
        url: http://wilson.ci.chef.co/job/chef-server-12-build/architecture=x86_64,platform=el-5,project=chef-server,role=builder/191/
        duration: 34m16s
      el-6:
        result: SUCCESS
        url: http://wilson.ci.chef.co/job/chef-server-12-build/architecture=x86_64,platform=el-6,project=chef-server,role=builder/191/
        duration: 32m2s
      el-7:
        result: SUCCESS
        url: http://wilson.ci.chef.co/job/chef-server-12-build/architecture=x86_64,platform=el-7,project=chef-server,role=builder/191/
        duration: 32m3s
      ubuntu-10.04:
        result: SUCCESS
        url: http://wilson.ci.chef.co/job/chef-server-12-build/architecture=x86_64,platform=ubuntu-10.04,project=chef-server,role=builder/191/
        duration: 28m51s
  chef-server-12-trigger-ad_hoc:
    result: SUCCESS
    url: http://wilson.ci.chef.co/job/chef-server-12-trigger-ad_hoc/114/
    duration: 9s
